src/main/java/mil/dds/anet/beans/search/ReportSearchQuery.java

```diff
@@ -15,6 +15,8 @@
 	Integer authorId;
 	DateTime engagementDateStart;
 	DateTime engagementDateEnd;
+	Integer engagementDayOfWeek;
+	Boolean includeEngagementDayOfWeek;
```

Comment:
I prefer to make new fields `final` (at some point [e.g. when we have decided on a Checkstyle profile] we should consider changing them all).